### PR TITLE
Filter to add custom panel block inside panel

### PR DESCRIPTION
**Pull request by nathanrodrigues2111**
### Description
Added filter to add custom panel block inside panel
Filter name : uagb.pluginSidebar

### Screenshots
- 

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
Checked if the  filter worked properly

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/extensions/spectra-page-settings-popup/index.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ import SettingsIcons from './icons.js';
 import PageCustomCSS from '../custom-page-css';
 import { PanelBody } from '@wordpress/components';
 
-const SpectraPageSettingsPopup = () => {
+const SpectraPageSettingsPopup = (props) => {
```
Comment (bsf-zanev) [inline src/blocks/extensions/spectra-page-settings-popup/index.js:12]: Please lint this code.

File: src/blocks/extensions/spectra-page-settings-popup/index.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ import SettingsIcons from './icons.js';
 import PageCustomCSS from '../custom-page-css';
 import { PanelBody } from '@wordpress/components';
```
Comment (bsf-zanev) [inline src/blocks/extensions/spectra-page-settings-popup/index.js:10]: Add `import { applyFilters } from '@wordpress/hooks';` here and replace `wp.hooks.applyFilters` with `applyFilters`;

File: src/blocks/extensions/spectra-page-settings-popup/index.js [hunk 3]
```diff
@@ -9,7 +9,8 @@ import SettingsIcons from './icons.js';
...
 
-const SpectraPageSettingsPopup = () => {
+const SpectraPageSettingsPopup = (props) => {
+	const pluginSidebar = wp.hooks.applyFilters('uagb.pluginSidebar', '', props);
```
Comment (bsf-zanev) [inline src/blocks/extensions/spectra-page-settings-popup/index.js:13]: Let's add two filters:
1. `spectra.page-sidebar.before`
2. `spectra.page-sidebar`

This is as per the standard of hook naming we're following from now on.
`spectra.**section-name**.**sub-section-name**.**position**`

File: src/blocks/extensions/spectra-page-settings-popup/index.js [hunk 4]
```diff
@@ -31,6 +32,7 @@ const SpectraPageSettingsPopup = () => {
 				title={ __( 'Spectra Page Settings' ) }
 				className={'spectra-sidebar'}
 			>
+				{pluginSidebar}
```
Comment (bsf-zanev) [inline src/blocks/extensions/spectra-page-settings-popup/index.js:35]: Add the before filter here, and the other filter after the Custom CSS PanelBody.

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
Please address these changes.

File: src/blocks/extensions/spectra-page-settings-popup/index.js (hunk 4 above)
Comment (nathanrodrigues2111) [reply src/blocks/extensions/spectra-page-settings-popup/index.js]: Resolved

File: src/blocks/extensions/spectra-page-settings-popup/index.js (hunk 3 above)
Comment (nathanrodrigues2111) [reply src/blocks/extensions/spectra-page-settings-popup/index.js]: Resolved

i have added

spectra.page-sidebar.before
spectra.page-sidebar.after

Just to keep things consistent

File: src/blocks/extensions/spectra-page-settings-popup/index.js (hunk 2 above)
Comment (nathanrodrigues2111) [reply src/blocks/extensions/spectra-page-settings-popup/index.js:10]: Resolved

File: src/blocks/extensions/spectra-page-settings-popup/index.js (hunk 1 above)
Comment (nathanrodrigues2111) [reply src/blocks/extensions/spectra-page-settings-popup/index.js]: Resolved

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**